docs/api/framework-routers/HydratedRouter.md

```diff
@@ -21,7 +21,7 @@ https://github.com/remix-run/react-router/blob/main/packages/react-router/lib/do
 ## Summary
 
 Framework-mode router component to be used to to hydrate a router from a
-`ServerRouter`. See [`entry.client.tsx`](../api/framework-conventions/entry.client.tsx).
+`ServerRouter`. See [`entry.client.tsx`](../framework-conventions/entry.client.tsx).
```

Comment:
@geonhwiii thanks for these fixes! This doc and `ServerRouter.md` are both autogenerated (see the note above).

Will you:
1. Correct the jsdoc in the source code referenced in the comment
2. Point this branch to `dev` instead of `main` since it'll be updating code technically